fusor-ember-cli/.gitignore

```diff
@@ -5,8 +5,10 @@
 /tmp
 
 # dependencies
-/node_modules
-/bower_components
+/node_modules*
+/bower_components*
+.nvmrc
```

Comment:
we should probably check in .nvmrc so that anyone can "nvm use" and be on the same version of node.  If they don't have nvm it doesn't hurt anything.